backend/src/main/java/de/bund/digitalservice/ris/norms/application/port/output/UpdateMigrationLogCompletedPort.java

```diff
@@ -0,0 +1,25 @@
+package de.bund.digitalservice.ris.norms.application.port.output;
+
+import de.bund.digitalservice.ris.norms.domain.entity.MigrationLog;
+import java.util.UUID;
+
+/**
+ * Port for updating the completion state of a migration log entry.
+ */
+public interface UpdateMigrationLogCompletedPort {
```

Comment:
`CompleteMigrationLogPort`? We only use it to set it to completed so we could keep the surface of the port smaller

Reply:
Agree